tools/README.md

```diff
@@ -5,34 +5,64 @@ This tool allows ballerina message broker developers to get an idea about the pe
 - Connection Load  - The number of message producers , or the number of concurrent connections a system can support.
 - Message throughput - The number of messages or message bytes that can be pumped through a messaging system per second.
 
-At the moment the tool allows developers to publish messages to queues and test the performance and get a report of the result.
+At the moment the tool allows developers to publish/consume messages to/from queues and topics and test the performance and get a report of the result.
 
-## How to use 
+# How to use
 
-1. Copy following jars to JMETER_HOME/lib.
+## Prerequisite
+
+1. Create two queues ```micro_benchmark_queue1``` and ```micro_benchmark_queue2```.The script uses these queues separately to study the performance of queue and topic operations.
+
+2. Copy following jars to JMETER_HOME/lib.
 
     - andes-client-0.13.wso2v8.jar
     - geronimo-jms_1.1_spec-1.1.0.wso2v1.jar
     - slf4j-1.5.10.wso2v1.jar
 
-2. Create a queue named micro_benchmark_queue in broker
+### Publish messages to queues/topics
 
-3. Include the following details in ballerina_message_broker_performance_test.properties file which is located at resources/.
+1. Include the following details in broker_performance_test_publisher.properties file which is located at resources/.
 ```properties
 jmeter_home = <jmeter_home>
-jmx_file_location = <jmx_file_location>
-jndi_file_location = <jndi_file_location>
 thread_count = <number_of_threads>
 message_size = <size_of_the_message>
 number_of_messages = <number_of_messages_need_to_be_published>
 throughput= <throughput_need_to_be_maintained>
 ```
-4.Run ```./broker_performance_test.sh <location_of_properties_file>```
+2.Run ```./broker_test_publisher.sh -p <location_of_properties_file> -d queue/topic```
 
 Upon completion of the test,you will be directed to a web interface which contains the summary of the results obtained by the test.
 
-![statistics.png](scripts/images/Statistics.png)
-![throughput.png](scripts/images/Throughput.png)
+### Consume messages from queues/topics
+
+1. Include the following details in broker_performance_test_consumer.properties file which is located at resources/.
```

Comment:
is this the same file that was used for the publisher? or is it different? What happens if both publisher and consumer are run without specifying a path for the properties file?